scripts/get_maintainer.py

```diff
@@ -197,17 +228,27 @@ def main():
     args = get_args()
     all_subsystems = parse_maintainers()
     paths = []
+    arglist = []
     downloads = []
+    split_patches = []
 
     for pr in args.github_pr or []:
         downloads += [download(pr)]
 
     for arg in args.arg + downloads:
+        if os.path.exists(arg):
+            patches = split_patchset(arg)
+            if patches:
+                split_patches += patches
+                continue
+        arglist += [arg]
```

Comment:
If `arg` is a single arg here in the loop? In that case I again suggest. If not, let it be.
```suggestion
        arglist.append(arg)
```

Reply:
Exactly, will change